rmgpy/tools/canteraModel.py

```diff
@@ -405,3 +405,137 @@ def findIgnitionDelay(time, yVar=None, metric='maxDerivative'):
         index = multdata.argmax()
         return time[index]
 
+
+def checkNearlyEqual(value1, value2, dE = 1e-5):
+    """
+    Check that two values are nearly equivalent by abs(val1-val2) < abs(dE*val1)
+    """
+    
+    if abs(value1-value2) <= abs(dE*value1):
+        return True
+    else:
+        return False
+    
+    
+def checkEquivalentCanteraSpecies(ctSpec1, ctSpec2, dE=1e-5):
+    """
+    Checks that the two cantera species are nearly equivalent
+    """
+    try:
+        assert ctSpec1.name == ctSpec2.name, "Identical name"
+        assert ctSpec1.composition == ctSpec2.composition, "Identical composition"
+        assert ctSpec1.size == ctSpec2.size, "Identical species size"
+        assert ctSpec1.charge == ctSpec2.charge, "Identical charge"
+
+        if ctSpec1.transport or ctSpec2.transport:
+            trans1 = ctSpec1.transport
+            trans2 = ctSpec2.transport
+
+            assert checkNearlyEqual(trans1.acentric_factor, trans2.acentric_factor, dE), "Identical acentric factor"
+            assert checkNearlyEqual(trans1.diameter, trans2.diameter, dE), "Identical diameter"
+            assert checkNearlyEqual(trans1.dipole, trans2.dipole, dE), "Identical dipole moment"
+            assert trans1.geometry == trans2.geometry, "Identical geometry"
+            assert checkNearlyEqual(trans1.polarizability, trans2.polarizability, dE), "Identical polarizibility"
+            assert checkNearlyEqual(trans1.rotational_relaxation, trans2.rotational_relaxation, dE), "Identical rotational relaxation number"
+            assert checkNearlyEqual(trans1.well_depth, trans2.well_depth, dE), "Identical well depth"
+
+        if ctSpec1.thermo or ctSpec2.thermo:
+            thermo1 = ctSpec1.thermo
+            thermo2 = ctSpec2.thermo
+
+            Tlist = [300,500,1000,1500,2000]
+            for T in Tlist:
+                assert checkNearlyEqual(thermo1.cp(T), thermo2.cp(T), dE),  "Similar heat capacity"
+                assert checkNearlyEqual(thermo1.h(T), thermo2.h(T), dE), "Similar enthalpy"
```

Comment:
Doing a relative comparison could lead to false failures, or even division by zero errors, when H is close to zero. Maybe this one (and any others that can be arbitrarily close to 0) should be absolute comparisons instead of relative ones?